config.yml

```diff
@@ -43,6 +43,11 @@ aws:
       PIPENV_VERSION: 2018.10.13
       PY_YAML_VERSION: 3.13
 
+awslnx-systemd:
+  "2":
+    template_vars:
+      VERSION: 2.0.20190508
```

Comment:
Can't you think about any test that this image does what we want ?

Reply:
This image does not do anything more that having systemd in it. To test if systemd is running, we would need to run it with SYS_ADMIN capability (and some other options). I do not know if Travis allow this and it would require further development of the main script

Reply:
Ah, yeah you're right. I just tested and couldn't go anywhere with systemctl.
At least I think you can check that pip and which are installed (maybe by doing a `which pip` 😋?).
We can't really test cronie with a `systemcl is-enabled` though (it was my test).